packages/features/src/settings/views/authorized-zkapps.tsx

```diff
@@ -0,0 +1,83 @@
+import { useEffect, useState } from "react"
+
+import { fetcher } from "@/common/lib/fetch"
+import { AppLayout } from "@/components/app-layout"
+import { SettingsPageLayout } from "@/components/settings-page-layout"
+import { X } from "lucide-react"
+
+type AuthorizedZkAppsViewProps = {
+  onCloseClicked: () => void
+}
+
+type ZkApp = { title: string; description: string; image: string; url: string }
+
+export const AuthorizedZkAppsView = ({
+  onCloseClicked,
+}: AuthorizedZkAppsViewProps) => {
+  const [connectedApps, setConnectedApps] = useState([] as ZkApp[])
+
+  useEffect(() => {
```

Comment:
Can we move any of the view's logic to the route? This is a general thing we tend to do, view should only have the template, route should have the logic.